core/src/main/java/nl/nn/adapterframework/http/HttpSender.java

```diff
@@ -577,7 +583,16 @@ public String getResponseBodyAsString(HttpResponseHandler responseHandler) throw
 			}
 			return headersXml.toXML();
 		}
-		String responseBody = responseHandler.getResponseAsString(true);
+		String responseBody = null;
+		try {
+			responseBody = responseHandler.getResponseAsString(true);
+		} catch(IOException e) {
+			if(throwIOExceptionWhenParsingResponse) {
+				throw e;
+			}
+			return null;
```

Comment:
And what happens to the exception then? It's just gone?
At least log it!

Reply:
This method will be called in case of an exception. The exception thrown by this method is not the actual cause. Therefore, it can be ignored. The caller methods will throw the actual exception

Reply:
I think it would be wise to add a comment describing the problem. Either near the try/catch or as javadoc above the method.

When an exception occurs and the response cannot be parsed, we do not want to throw a 'missing response' exception. Since this method is used when handling other exceptions (link to wss exception here) (link to http exception here), silently return null, to avoid npe's, ioexceptions etc.